src/design-system-team.md.njk

```diff
@@ -14,6 +14,7 @@ If you want to contact the team you can
 
 ## GOV.UK Design System team
 - Kelly Lee – Delivery Manager
+- Steve Messer - Senior Product Manager
```

Comment:
I _think_ that dash is the normal dash when it needs to be the em dash

```suggestion
- Steve Messer – Senior Product Manager
```

Reply:
I just noticed this, thanks for fixing!